pkg/configprovider/opampprovider/provider.go

```diff
@@ -38,6 +39,7 @@ type ConfigFragment struct {
 	Extensions struct {
 		OpAmp struct {
 			RemoteConfigurationDirectory string `yaml:"remote_configuration_directory"`
+			NewConfigMergeFlowDisabled bool `yaml:"disable_tag_new_config_merge"`
```

Comment:
In provider.go, since the name got changed to ```disable_tag_new_config_merge```, shouldn't the variable also be changed to DisableTagNewConfigMerge (next to bool)? Just for consistency 😆 

```
	Extensions struct {
		OpAmp struct {
			RemoteConfigurationDirectory string `yaml:"remote_configuration_directory"`
			NewConfigMergeFlowDisabled bool `yaml:"disable_tag_new_config_merge"`
		} `yaml:"opamp"`
	} `yaml:"extensions"`
```

Reply:
Done @chan-tim-sumo